fabric-renderer-api-v1/src/main/java/net/fabricmc/fabric/api/renderer/v1/material/BlendMode.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.renderer.v1.material;
+
+import net.minecraft.block.BlockRenderLayer;
+
+public enum BlendMode {
+	DEFAULT(null),
+	SOLID(BlockRenderLayer.field_9178),
+	CUTOUT_MIPPED(BlockRenderLayer.CUTOUT_MIPPED),
+	CUTOUT(BlockRenderLayer.field_9174),
+	TRANSLUCENT(BlockRenderLayer.field_9179);
```

Comment:
May this be a fake enum given block render layers are no longer enums?

Reply:
This is fully within our control and not likely to grow without accompanying breaking changes caused by Mojang.  Happy with enum for now.

Reply:
K, in 19w39a this happens
![image](https://user-images.githubusercontent.com/7806504/65810181-9a1fbe80-e16c-11e9-9a04-a366c75b6688.png)
9 types of new render layers, thanks mojang. Guess we need to hold off on bumping api patch versions then :sweat_smile:

Reply:
Likely their attempt to fix the render issue where water would draw over smoke particles ect.